package.json

```diff
@@ -39,6 +39,8 @@
     "jest-junit": "^6.3.0",
     "metro-react-native-babel-preset": "^0.52.0",
     "node-fetch": "^2.3.0",
+    "patch-package": "^6.2.0",
+    "postinstall-prepare": "^1.0.1",
```

Comment:
Do you have more insights into why this is needed? The repo has [an explanation](https://github.com/ds300/patch-package#why-use-postinstall-postinstall-with-yarn) but I still don't understand why we need to run `prepare` after removing a dependency.

Reply:
`node_modules` are regenerated when I run `yarn remove xyz` and the patch will not be applied w/o `postinstall-prepare` package.

To test it, please checkout to this branch, run `yarn remove postinstall-prepare` and check `node_modules/react-native/React/Views/RCTShadowView.m:159` file.  You should have the original version w/o the patch. When you add`postinstall-prepare` and run i.e `yarn remove flow-bin` the patch is applied and this is the expected result.

I think it is necessary since `yarn remove` is a regular command that is used and we want to have `react-native` patched after that.

Reply:
That makes sense, thanks. It seems odd that yarn would do that 😞